instruments/keithley/keithley485.py

```diff
@@ -413,10 +413,16 @@ def _parse_measurement(self, measurement):
 
         try:
             status = self.Status(status)
-            if status != self.Status.normal:
-                raise ValueError("Instrument not in normal mode: {}".format(status.name))
-            if function != b"DC":
-                raise ValueError("Instrument not returning DC function: {}".format(function))
+        except ValueError:
+            raise ValueError(f"Invalid status word in measurement: {status}")
+
+        if status != self.Status.normal:
+            raise ValueError("Instrument not in normal mode: {}".format(status.name))
+
+        if function != b"DC":
+            raise ValueError("Instrument not returning DC function: {}".format(function))
+
+        try:
```

Comment:
The previous statement tried to get the status, then tried to see what status it is in a `try` statement. If one of the `if` statements inside the `try` pass, they will raise an error, which would jump to the except and raise another error message than the one that should be raised. 
Now trying to read to get the statusword from the respective enum class first. If not possible, then raise a respective error. After that we can go into the if statements and check the constraints, if they fail, they will now return the correct error message.